src/main/java/frc/robot/commands/ShooterOffCommand.java

```diff
@@ -25,7 +25,7 @@ public void initialize() {
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    m_shooterSubsystem.setFlywheel(0);
+    m_shooterSubsystem.setFlywheelPower(0);
```

Comment:
Is there a reason why this can't go in initialize?

Reply:
No clue, it wasn't there before.  Could make this an instant command?

Reply:
@TiaSinghania Thoughts?